docs/docs/using-cloudinary-image-service.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Using Cloudinary image service for media optimization  
+---
+Cloudinary is a cloud-based end-to-end media management platform that provides solutions to help site creators serve optimized media files(images and videos) to their audiences. It also provides a lot of optional transformations that can be carried out on these media assets.  
+
+In this guide you will take a look at the  [gatsby-source-cloudinary](/packages/gatsby-source-cloudinary/) and [gatsby-transformer-cloudinary](/packages/gatsby-transformer-cloudinary/) which are used to improve the experience of handling images on Gatsby sites.
+
+Plugins are generally used to abstract functionality in Gatsby. In this case `gatsby-source-cloudinary` plugin is a [source plugin](/docs/creating-a-source-plugin/) which helps to connect Cloudinary media storage capabilities to your site.  
+
+If you already have some images hosted on Cloudinary and would like to leverage on the plugin system of Gatsby this plugin is for you. 
+
+> Here's a [demo site that uses the gatsby-source-cloudinary](https://gsc-sample.netlify.com) showcasing  optimized images in a masonry grid, served from Cloudinary
+
+## The problem with handling images on the web
+
+Dealing with images on the web has always been a problem as unoptimized images can slow down your site. The processes put in place to create the best media experience can take a lot of time to implement. 
+
+## Solutions Cloudinary provides 
+
+Cloudinary provides a couple of amazing solutions to this problem , namely: 
+- Remote storage and delivery of images via CDN
+- Wider range of transformations over [gatsby-image](/docs/using-gatsby-image/).
```

Comment:
```suggestion
- Wider range of transformations over [gatsby-image](/docs/using-gatsby-image/).
```
What do we mean by over?

Reply:
The Cloudinary plugin offers a wider range of transformations than gatsby-images

Reply:
I'd say that instead. I read over as using the two in combination allowed you to get more out of gatsby-image.